salt_stack/salt/top.sls

```diff
@@ -1,3 +1,5 @@
+{% set use_scalyr = salt['pillar.get']('scalyr:log_key', False) != False %}
```

Comment:
Would it be better to put this check in the scalyr formulas?

Reply:
Yes, that'd make top.sls a lot cleaner. Not sure if it'll work to effectively blank out the entire formula though. Will spend a little more time and try.